runtime/src/main/java/org/corfudb/runtime/LogReplicationRoutingQueueListener.java

```diff
@@ -92,16 +97,20 @@ public final void onNext(CorfuStreamEntries results) {
         }
 
         boolean callbackResult = false;
+        List<Queue.RoutingTableEntryMsg> entries =
+                results.getEntries().entrySet().stream().map(CSEntry ->
+                        (Queue.RoutingTableEntryMsg)CSEntry.getValue().get(0).getPayload()).collect(Collectors.toList());
```

Comment:
there can be multiple RoutingTableEntryMsg in a single CorfuStreamEntry.  We need to collect all of them